app/scripts/templates/sign_in_bounced.mustache

```diff
@@ -0,0 +1,20 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-signin-bounced-header">{{#t}}Sorry. We've locked your account.{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="graphic graphic-mail-bounced"></div>
+
+    <p>{{#t}}The confirmation email we sent to %(email)s was returned and we've locked your account to protect your Firefox data.{{/t}}</p>
+
+    <div class="extra-links">{{#unsafeTranslate}}If this is a valid email address, <a href="https://support.mozilla.org/" target="_blank">let us know</a> and we can help unlock your account.{{/unsafeTranslate}}</div>
```

Comment:
To simplify the burden on the l10n team, can you change 

`href="https://support.mozilla.org/" target="_blank"` 
to 
`%(escapedSupportLinkAttrs)s` 

and then pass in `escapedSupportLinkAttrs` in [setInitialContext](https://github.com/mozilla/fxa-content-server/pull/5318/files#diff-5c3e2c27ed2fcc4958e548f3e8821dd1R30)? We have found doing so allows us to add new/modify existing query parameters when they are needed w/o causing a string update and a string re-translation. Since none of the parameters obviously have no XSS vectors, not escaping should be fine.

While we are on the link, is there supposed to be a link to a specific article? If we send users to SUMO, will they actually be able to help users unlock their account?